src/clients/storage/BlobStorageClient.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+let request = require('request');
+const apiUrlBase = process.env.FORTIS_CENTRAL_ASSETS_HOST || 'https://fortiscentral.blob.core.windows.net/';
```

Comment:
Let's declare the default without trailing slash so that the url construction below is clearer.